tests/unit/introspection.tcl

```diff
@@ -325,6 +325,36 @@ start_server {tags {"introspection"}} {
         }
     }
 
+    test {CLIENT SETINFO can set a library name to this connection} {
+        r CLIENT SETINFO libname redis.py
+        r CLIENT SETINFO libver 1.2.3
+        r client info
+    } {*lib-name=redis.py lib-ver=1.2.3*}
```

Comment:
It's slightly annoying that you set "LIBNAME" but it get's printed back as "LIB-NAME", I would prefer everything being "LIB-NAME".

Reply:
i agree, but all tokens today are without hyphens, and all CLIENT INFO field are with them.
maybe we can fold this discussion into the one above, and delete all of these (just document `key` and `value`).

Reply:
discussed with Yossi, we'll add a hyphen in the tokens (apparently that's already the case in the JSON files)